projects/kit/components/input-password/input-password.component.ts

```diff
@@ -44,6 +50,8 @@ import {InputPasswordOptions, TUI_INPUT_PASSWORD_OPTIONS} from './input-password
             useExisting: forwardRef(() => TuiInputPasswordComponent),
         },
         HINT_CONTROLLER_PROVIDER,
+        TuiDestroyService,
```

Comment:
There's no need for explicit destroy service, MODE_PROVIDER already has it baked in

Reply:
Did you mean HINT_CONTROLLER_PROVIDER? I can't find TuiDestroyService inside MODE_PROVIDER :(

Reply:
Right, I meant HINT_CONTROLLER_PROVIDER, sorry.